src/Sql/Sql/Database Backup/Cmdlet/RestoreAzureRMSqlDatabase.cs

```diff
@@ -330,6 +330,14 @@ public class RestoreAzureRmSqlDatabase
             HelpMessage = "The federated client id for the SQL Database. It is used for cross tenant CMK scenario.")]
         public Guid? FederatedClientId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the encryption protector key auto rotation status
+        /// </summary>
+        [Parameter(Mandatory = false,
+        ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The AKV Key Auto Rotation status")]
+        public bool? EncryptionProtectorAutoRotation { get; set; }
```

Comment:
why not use a **SwitchParameter** instead?

Reply:
changed to switch parameter